cmake/remillConfig.cmake.in

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2017 Trail of Bits, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.2)
+
+if(WIN32)
+  set(REMILL_LIBRARY_LOCATION "${CMAKE_INSTALL_PREFIX}/remill/lib/remill.lib")
+  set(REMILL_INCLUDE_LOCATION "${CMAKE_INSTALL_PREFIX}/remill/include")
+else()
+  set(REMILL_LIBRARY_LOCATION "${CMAKE_INSTALL_PREFIX}/lib/libremill.a")
```

Comment:
That would be great to implement! Right now you can only expect to install one version system-wide (in my opinion it should match the LLVM version available in the package repository).

You can still install more than one version using prefixes, and then essentially use the same method we use to find_package LLVM.

I am adding this in the todo list though, so that we can implement this later!

Reply:
Is there a way for us to prepare for this now, e.g. by making `libremill.a` a symlink to `libremill-M.m.a`?

Reply:
Or maybe having an install directory version number would make sense, with symlinks? You are ultimately the best person to decide on how this should all work. In theory, one might want to eventually have a `find_package` for CMake allow the Remill version to be specified.

Reply:
I still have to think about this, as we also have other things to keep in mind (like the semantics path or the compatibility with mcsema). Maybe I can do some tests and post the results!